src/display/display_utils.js

```diff
@@ -693,21 +687,25 @@ class OutputScale {
     return globalThis.devicePixelRatio || 1;
   }
 
-  static getCappedWindowArea(capAreaFactor) {
-    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) {
-      return Math.ceil(
-        window.innerWidth *
-          window.innerHeight *
-          this.pixelRatio ** 2 *
-          (1 + capAreaFactor / 100)
-      );
+  static capPixels(maxPixels, capAreaFactor) {
+    if (capAreaFactor >= 0) {
+      const winPixels =
+        typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")
```

Comment:
could we simplify into `Math.ceil((typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING") ? window.innerWidth * window.innerHeight  :  window.screen.availWidth * window.screen.availHeight)...` ?

Reply:
Sure, we can do that. (I'm only a little bit worried that it might be less readable.)